gwtlib/display.py

```
@@ -272,57 +255,50 @@ def print_branch(b, kind):
-            pass
+    # Get local branches (single fetch, used for both collection and categorization)
+    local_branches = set()
+    try:
+        result = run_git_command(
+            ["for-each-ref", "--format=%(refname:short)", "refs/heads/"], git_dir
+        )
+        for branch in result.stdout.strip().split("\n"):
+            if branch:
+                local_branches.add(branch)
+    except Exception:
+        pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)